File: src/main/java/gg/grounds/repo/MaintenanceWhitelistRepository.java
package gg.grounds.repo;

import gg.grounds.domain.model.MaintenanceWhitelistEntity;
import io.quarkus.hibernate.reactive.panache.PanacheRepositoryBase;
import jakarta.enterprise.context.ApplicationScoped;

import java.util.UUID;

@ApplicationScoped
public class MaintenanceWhitelistRepository implements PanacheRepositoryBase<MaintenanceWhitelistEntity, UUID> {}

File: src/main/kotlin/gg/grounds/repo/MaintenanceWhitelistRepository.kt
package gg.grounds.repo

import gg.grounds.domain.model.MaintenanceWhitelistEntity
import io.quarkus.hibernate.reactive.panache.PanacheRepositoryBase
import jakarta.enterprise.context.ApplicationScoped
import java.util.*

@ApplicationScoped
class MaintenanceWhitelistRepository : PanacheRepositoryBase<MaintenanceWhitelistEntity?, UUID?>
